frontend/app/src/store/blockchain/tokens.ts

```diff
@@ -102,6 +104,20 @@ export const useBlockchainTokensStore = defineStore('blockchain/tokens', () => {
     await fetchDetectedTokens();
   });
 
+  const { isTaskRunning } = useTasks();
+  const isDetecting = isTaskRunning(TaskType.FETCH_DETECTED_TOKENS);
+
+  const { fetchBlockchainBalances } = useBlockchainBalancesStore();
+
+  watch(isDetecting, async (curr, prev) => {
```

Comment:
do you think we can adjust the names here to isDetecting & wasDetecting for clarity?